acme/order.go

```diff
@@ -296,16 +300,16 @@ func canonicalize(csr *x509.CertificateRequest) (canonicalized *x509.Certificate
 }
 
 // ipsAreEqual compares IPs to be equal. IPv6 representations of IPv4
-// adresses are NOT considered equal to the IPv4 address in this case.
-// Both IPs should be the same version AND equal to each other.
+// adresses are considered equal to the IPv4 address in this case.
+// TODO: is this behavior OK to keep?
```

Comment:
What about this behavior exactly are we questioning? Current behavior (as written) is that they will NOT be considered equal. Is that correct? I could be misunderstanding the code. My interpretation conflicts with the current comment above the function.